M2/Macaulay2/editors/README.md

```diff
@@ -0,0 +1,24 @@
+# Macaulay2 editors
+
+This directory contains template files used by the
+[generateGrammar](https://macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Style/html/_generate__Grammar.html) function in the *Style* package to generate
+grammar files used by various editors for syntax highlighting and automatic
+completion of Macaulay2 code.
+
+Each subdirectory contains one or more template files for a particular
+application.
+
+* [`Macaulay2Web`](Macaulay2Web): Symbols for the Macaulay2 web app.
```

Comment:
What's the point of this one? The file is basically empty.

Reply:
Maybe the Macaulay2Web repository should have this, too?

Reply:
I think it's for autocompletion -- it's just a list of all the the symbols.

Cc: @pzinn